dietpi/dietpi-software

```diff
@@ -7971,6 +7980,17 @@ _EOF_
 
 		fi
 
+		software_id=201 # Tailscale
+		if (( ${aSOFTWARE_INSTALL_STATE[$software_id]} == 1 )); then
+
+			Banner_Installing
+			G_EXEC curl -fsSL https://pkgs.tailscale.com/stable/debian/bullseye.noarmor.gpg -o /usr/share/keyrings/tailscale-archive-keyring.gpg
```

Comment:
there you might need to detect the Debian version as we support Buster, Bullseye as well as Bookworm